playground/ci/find_beam_examples.py

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def find_examples(work_dir: str) -> List[str]:
+    """Find examples files
+    Find all files which contain the beam-playground tag.
+
+    Args:
+        work_dir: directory where to search examples.
+
+    Returns:
+        List of paths to all tagged files.
+    """
+    return []
```

Comment:
why is it empty?

Reply:
This method should be implemented with [BEAM-13135].